week-14/practices/05-Navigation/README.md

```diff
@@ -1,22 +1,153 @@
 # Practice: Navigation
 
-In this practice you will learn how to create navigation for your Route
-paths using `Link` and `NavLink`.
+In this practice you will learn how to create navigation for your Route paths
+using `Link` and `NavLink`.
 
 ## Phase 0: Setup
 
-Download the [starter][starter]. `cd` into the folder and run the `npm install`
-command in your terminal.
+Download the [starter][starter]. `cd` into the folder and run the `npm install`command in your terminal.
 
 Next run `npm start` and check your browser to see if your code is running
 properly.
 
 Check your browser. You should only see an `h1` saying App Components.
```

Comment:
**Line 13:** "Check your browser." Feels redundant since it was just said in the last sentence

Reply:
edited